Velaptor/Services/FontMetaDataParseResult.cs

```diff
@@ -0,0 +1,55 @@
+﻿// <copyright file="FontMetaDataParseResult.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.Services
+{
+    /// <summary>
+    /// Holds information after the result of parsing font meta data from font file paths
+    /// using the <see cref="FontMetaDataParser"/>.
+    /// </summary>
+    internal readonly struct FontMetaDataParseResult
+    {
+        /// <summary>
+        /// Gets a value indicating if the string contains the metadata.
+        /// </summary>
+        public readonly bool ContainsMetaData;
+
+        /// <summary>
+        /// Gets a value indicating whether or not the meta data in the string is valid.
+        /// </summary>
+        public readonly bool IsValid;
+
+        /// <summary>
+        /// Gets the data before the metadata section.
+        /// </summary>
+        public readonly string MetaDataPrefix;
+
+        /// <summary>
+        /// Gets the meta data string.
```

Comment:
Refactor as below.  **meta data** should be one word

```suggestion
        /// Gets the metadata string.
```

Search entire code base for **meta data** and replace with **metadata**